generators/generatorPickerFunctions.cc

```diff
@@ -294,7 +317,7 @@ bool uniformMassExponentialTPicker::operator() (double& invariantMass, double& t
 		}
 		restarts++;
 		if(restarts >= 5 && not done) {
-                            printErr << restarts << " attempts to generate t' failed." << endl;
```

Comment:
not important, but maybe you can replace
&& -> and
|| -> or
for all the generator source files